libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/components/dialogs/ConfirmationDialog.kt

```diff
@@ -39,18 +37,20 @@ import io.element.android.libraries.ui.strings.R as StringR
 fun ConfirmationDialog(
     title: String,
     content: String,
+    onSubmitClicked: () -> Unit,
+    onDismiss: () -> Unit,
```

Comment:
These 2 lambdas should never be null, as we will *always* want the visible buttons in some dialog to trigger an action.